docs/deployment.rst

```diff
@@ -165,62 +165,35 @@ The timeout for an operation on the Kafka store. This is the maximum time that a
 * Importance: medium
 
 Kafka & ZooKeeper
-~~~~~~~~~~~~~~~~~
+-----------------
 
 Please refer to :ref:`schemaregistry_operations` for recommendations on operationalizing Kafka and ZooKeeper.
 
-.. _schemaregistry_mirroring:
+Migration from Zookeeper master election to Kafka master election
+-----------------------------------------------------------------
 
-Multi-DC Setup
-~~~~~~~~~~~~~~
+It is not required to migrate from Zookeeper based election to Kafka based master election. If
+you choose to do so, you need to make the below outlined config changes as the first step in all
+Schema Registry nodes
 
-Overview
-^^^^^^^^
-Spanning multiple datacenters with your Schema Registry provides additional protection against data loss and improved latency. The recommended multi-datacenter deployment designates one datacenter as "master" and all others as "slaves". If the "master" datacenter fails and is unrecoverable, a "slave" datacenter will need to be manually designated the new "master" through the steps in the Run Book below.
+- Remove ``kafkastore.connection.url``
+- Remove ``schema.registry.zk.namespace`` if its configured
+- Configure ``kafkastore.bootstrap.servers``
+- Configure ``schema.registry.group.id`` if you originally had ``schema.registry.zk.namespace`` for multiple Schema Registry clusters
 
+Downtime for Writes
+^^^^^^^^^^^^^^^^^^^^
 
-Recommended Deployment
-^^^^^^^^^^^^^^^^^^^^^^
+You can migrate from Zookeeper based master election to Kafka based master election by following
+below outlined steps. These steps would lead to Schema Registry not being available for writes
+for a brief amount of time.
 
-.. image:: multi-dc-setup.png
+- Make above outlined config changes on that node and also ensure ``master.eligibility`` is set to false in all the nodes
+- Do a rolling bounce of all the nodes.
+- Configure ``master.eligibility`` to true on the nodes that can be master eligible and bounce them
 
-In the image above, there are two datacenters - DC A, and DC B. Each of the two datacenters has its own ZooKeeper
-cluster, Kafka cluster, and Schema Registry cluster. Both Schema Registry clusters link to Kafka and ZooKeeper in DC A. Note that the Schema Registry instances in DC B have ``master.eligibility`` set to false, meaning that none can ever be elected master.
-
-To protect against complete loss of DC A, Kafka cluster A (the source) is replicated to Kafka cluster B (the target). This is achieved by running the :ref:`Replicator` <connect_replicator>` local to the target cluster.
-
-Important Settings
+Complete downtime
```

Comment:
i would just omit this -- it should be clear given `master.eligibility` being `true` after the previous bounce